src/etc/inc/interfaces.inc

```diff
@@ -213,7 +213,7 @@ function interface_vlan_configure(&$vlan)
         mwexecf('/usr/sbin/ngctl name %s: %s', array($tmpvlanif, $vlanif));
     }
 
-    legacy_vlan_tag($vlanif, $if, $vlan['tag']);
+    legacy_vlan_tag($vlanif, $if, $vlan['tag'], $vlan['pcp']);
```

Comment:
Make sure this doesn't cause a warning for old configs where the value is not set